specifications/secure-launch-specification.rst

```diff
@@ -354,22 +354,26 @@ This is the main configuration entry, providing the necessary information to
 invoke the DLE Handler and for the DLE Handler to invoke the DL.
 
 :tag: SLR_ENTRY_DL_INFO
-:bl_context: Allows the bootloader to provide a reference to a context object.
-:dl_handler: The address to the entry point for the DLE Handler.
-:dce_base: The base address where the DCE is located.
 :dce_size: The size of the DCE.
+:dce_base: The base address where the DCE is located.
+:dlme_size: The size of the DLME.
+:dlme_base: The base address where the DLME is located.
 :dlme_entry: The address for the entry point of the DLME.
```

Comment:
This is used as an offset from base instead of full address in https://github.com/TrenchBoot/grub/blob/grub-sl-2.12-v9/grub-core/loader/slaunch/txt.c#L1154, could you update the description?

Reply:
Yes I will fix that